src/tree/AzureAccountTreeItem.ts

```diff
@@ -3,15 +3,46 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { AzureAccountTreeItemBase, ISubscriptionContext } from 'vscode-azureextensionui';
+import { pathExists } from 'fs-extra';
+import { join } from 'path';
+import { workspace } from 'vscode';
+import { AzExtTreeItem, AzureAccountTreeItemBase, IActionContext, ISubscriptionContext } from 'vscode-azureextensionui';
+import { configFileName } from '../constants';
+import { LocalProjectTreeItem } from './localProject/LocalProjectTreeItem';
 import { SubscriptionTreeItem } from './SubscriptionTreeItem';
 
-export class AzureAccountTreeItem extends AzureAccountTreeItemBase {
+export class AzureAccountTreeItemWithProject extends AzureAccountTreeItemBase {
     public constructor(testAccount?: {}) {
         super(undefined, testAccount);
     }
 
     public createSubscriptionTreeItem(root: ISubscriptionContext): SubscriptionTreeItem {
         return new SubscriptionTreeItem(this, root);
     }
+
+    public async loadMoreChildrenImpl(clearCache: boolean, context: IActionContext): Promise<AzExtTreeItem[]> {
+        const children: AzExtTreeItem[] = await super.loadMoreChildrenImpl(clearCache, context);
+
+        for (const workspaceFolder of workspace.workspaceFolders || []) {
+            const workspaceFolderPath: string = workspaceFolder.uri.fsPath;
+            const configFilePathExists: boolean = await pathExists(join(workspaceFolderPath, configFileName));
+
+            if (configFilePathExists) {
+                const localProjectTreeItem: LocalProjectTreeItem = new LocalProjectTreeItem(this, workspaceFolderPath);
```

Comment:
Can we put all of this UI behind a feature flag? I have concerns, but I feel like they don't need to be ironed out in the first PR. I think this is a big feature that will have a ton of PRs and changes between now and when it's final. Using a feature flag would give me peace of mind to have this merged without trying to block on things 🙂